src/Bridges/Bridges.jl

```diff
@@ -23,17 +23,111 @@ include("debug.jl")
 """
     full_bridge_optimizer(model::MOI.ModelLike, ::Type{T}) where {T}
 
-Returns a `LazyBridgeOptimizer` bridging `model` for every bridge defined in
-this package and for the coefficient type `T`.
+Returns a [`LazyBridgeOptimizer`](@ref) bridging `model` for every bridge
+defined in this package (see below for the few exceptions) and for the
+coefficient type `T` in addition to the bridges in the list returned by
+`MOI.get(model, MOI.Bridges.ListOfNonstandardBridges{T}())`.
+
+See also [`ListOfNonstandardBridges`](@ref).
+
+!!! note
+    The following bridges are not added by `full_bridge_optimizer` except if
+    they are in the list returned by `MOI.get(model, MOI.Bridges.ListOfNonstandardBridges{T}())`
+    (see the docstrings of the corresponding bridge for the reason they are not
+    added):
+    * [`Constraint.SOCtoNonConvexQuadBridge`](@ref),
+      [`Constraint.RSOCtoNonConvexQuadBridge`](@ref) and
+      [`Constraint.SOCtoPSDBridge`](@ref).
+    * The subtypes of [`Constraint.AbstractToIntervalBridge`](@ref) (i.e.
+      [`Constraint.GreaterToIntervalBridge`](@ref) and
+      [`Constraint.LessToIntervalBridge`](@ref)) if `T` is not a subtype of
+      `AbstractFloat`.
 """
 function full_bridge_optimizer(model::MOI.ModelLike, ::Type{T}) where {T}
     bridged_model = LazyBridgeOptimizer(model)
+    for BT in MOI.get(model, ListOfNonstandardBridges{T}())
+        add_bridge(bridged_model, BT)
+    end
     Variable.add_all_bridges(bridged_model, T)
     Constraint.add_all_bridges(bridged_model, T)
     Objective.add_all_bridges(bridged_model, T)
     return bridged_model
 end
 
+"""
+    ListOfNonstandardBridges{T}() <: MOI.AbstractOptimizerAttribute
+
+Any optimizer can be wrapped in a [`LazyBridgeOptimizer`](@ref) using
+[`full_bridge_optimizer`](@ref). However, by default [`LazyBridgeOptimizer`](@ref)
+uses a limited set of bridges that are:
+
+  1. implemented in `MOI.Bridges`
+  2. generally applicable for all optimizers.
+
+For some optimizers however, it is useful to add additional bridges, such as
+those that are implemented in external packages (e.g., within the solver package
+itself) or only apply in certain circumstances (e.g.,
+[`Constraint.SOCtoNonConvexQuadBridge`](@ref)).
+
+Such optimizers should implement the `ListOfNonstandardBridges` attribute to
+return a vector of bridge types that are added by [`full_bridge_optimizer`](@ref)
+in addition to the list of default bridges.
+
+Note that optimizers implementing `ListOfNonstandardBridges` may be unusable
+if the non-standard bridges are not added.
```

Comment:
I know I wrote this, but it might be too strong language.

```
Note that optimizers implementing `ListOfNonstandardBridges` may require
package-specific functions or sets to be used if the non-standard bridges
are not added. Therefore, you are recommended to use
`model = MOI.instantiate(Package.Optimizer; with_bridge_type = T)` instead of 
`model = MOI.instantiate(Package.Optimizer)`.
```

Reply:
Indeed, it's better to point to instantiate